### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Layrovell**
[DEMO LINK](https://Layrovell.github.io/react_static-list-of-posts/)

### CODE REVIEW

**Review by sasha39612 — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+import { PostList } from './components/PostList/PostList';
+
+const preparedArray = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId),
+  postComments: comments.filter(comment => comment.postId === post.id),
```
Comment (aliceblunt8) [inline src/App.js:13]:
```suggestion
  comments: comments.filter(comment => comment.postId === post.id),
```

File: src/components/Type/type.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  phone: PropTypes.string.isRequired,
+  website: PropTypes.string.isRequired,
+  company: PropTypes.objectOf(PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    catchPhrase: PropTypes.string.isRequired,
+    bs: PropTypes.string.isRequired,
+  })),
+});
+
+export const CommentsPropTypes = PropTypes.shape({
```
Comment (aliceblunt8) [inline src/components/Type/type.js:27]:
```suggestion
export const CommentPropTypes = PropTypes.shape({
```

File: src/components/Type/type.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  postId: PropTypes.number.isRequired,
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  email: PropTypes.string.isRequired,
+  body: PropTypes.string.isRequired,
+});
+
+export const PostsPropTypes = PropTypes.shape({
```
Comment (aliceblunt8) [inline src/components/Type/type.js:35]:
```suggestion
export const PostPropTypes = PropTypes.shape({
```

**Review by aliceblunt8 — APPROVED** [review]
gj

**Pull request closed without merging**